docs/peggy/Peggy-Ceth-Account-Set.md

```diff
@@ -20,7 +22,9 @@ Similar to the account setting method, the transaction is priviledged and only t
 It will transfer the cEth from ethbridge module to an specific account.
 
 1. command format
-sifnodecli tx ethbridge rescue_ceth [cosmos-sender-address] [ceth_receiver_account] [ceth_amount]
+```
+sifnodecli tx ethbridge rescue_ceth $clp_admin_address $ceth-receiver-account $ceth_amount
+```
 
 2. command example
 sifnodecli tx ethbridge rescue_ceth sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd  10000000 --from=sif --yes
```

Comment:
can we remove this example and replace it with the real-world runbook command we will actually run? as above i think?

Reply:
Fine by me. It should look like this:

```
sifnodecli tx ethbridge rescue_ceth $sender_address $ceth-receiver-account $ceth_amount
```

Reply:
@ElliotFriedman to confirm, the first param is the sender - who do you mean by this? the admin? in that case does the following look correct?

sifnodecli tx ethbridge rescue_ceth $admin_address $ceth-receiver-account $ceth_amount

Reply:
@musnit same comment as before. Please update all the docs with your proposed changes from the comment and we should be good to merge.